ISISNeutronMuon/MDANSE
# 3D selection update

**Description of work**
Initial implementation of the 3d selection into MDANSE. Added atom name and fullname to the selection options. The invert selection was removed as it would make the updating from the 3d view to the checkbox and comboboxes more difficult.


![Animation17](https://github.com/ISISNeutronMuon/MDANSE/assets/14276033/184298f2-3118-456c-809f-0fc674dd0ce3)


**To test**
Load a trajectory, open the atom selector, and try out the selection of atoms in the 3d view. The checkbox, comboboxes, and textboxes should update. Try changing the selection in the checkbox and combobox selection the 3d view should update. Test out a few different things and click the use settings and run the job. Try testing out the atom charge and atom transmutation as well.
